src/components/Todo/todo.jsx

```
@@ -0,0 +1,241 @@
+import React, { Component } from 'react';
+import { List, Subheader, Tabs, Tab, Paper, TextField, Divider } from 'material-ui';
+import TodoItem from './todoItem';
+/* eslint react/jsx-no-bind: 0 */  // alternative to to re-write in all arrow functions
+
+const styles = {
+  tabHeadline: {
+    fontSize: 12,
+    paddingTop: 2,
+    marginBottom: 2,
+    fontWeight: 200,
+  },
+  listStyle: {
+    listStyleType: 'none',
+    margin: 0,
+    padding: 0,
+  },
+  divStyle: {
+    display: 'flex',
+    flexDirection: 'column',
+    background: '#75a3a3',
+    width: '100%',
+    height: '100%',
+  },
+  textField: {
+    width: '100%',
+  },
+  expand: {
+    width: '100%',
+    height: '100%',
+  },
+  dragHandle: {
+    width: '100%',
+    height: '15px',
+    background: 'grey',
+  },
+};
+
+const status = {
+  ALL: 'all',
+  ACTIVE: 'active',
+  COMPLETED: 'completed',
+};
+const ENTER_KEY = 13;
+
+class Todo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      editing: null,
+      newTodo: '',
+      todos: [],
+      storageKey: this.constructor.name,
+    };
+
+    this.componentWillUnmount = this.componentWillUnmount.bind(this);
+  }
+
+  // Life cycle methods
+  componentDidMount() {
+    /* global localStorage */
+    const savedState = localStorage.getItem(this.state.storageKey);
+    if (savedState !== null) {
+      /* eslint-disable-next-line */
```

We should _never_ just suppress all errors.

Why are you disabling all errors on this line?